crates/bevy_core_pipeline/src/tonemapping/mod.rs

```diff
@@ -128,35 +227,107 @@ pub fn queue_view_tonemapping_pipelines(
     pipeline_cache: Res<PipelineCache>,
     mut pipelines: ResMut<SpecializedRenderPipelines<TonemappingPipeline>>,
     upscaling_pipeline: Res<TonemappingPipeline>,
-    view_targets: Query<(Entity, &Tonemapping)>,
+    view_targets: Query<(Entity, Option<&Tonemapping>, Option<&Dither>), With<ViewTarget>>,
 ) {
-    for (entity, tonemapping) in view_targets.iter() {
-        if let Tonemapping::Enabled { deband_dither } = tonemapping {
-            let key = TonemappingPipelineKey {
-                deband_dither: *deband_dither,
-            };
-            let pipeline = pipelines.specialize(&pipeline_cache, &upscaling_pipeline, key);
-
-            commands
-                .entity(entity)
-                .insert(ViewTonemappingPipeline(pipeline));
-        }
+    for (entity, tonemapping, dither) in view_targets.iter() {
+        let key = TonemappingPipelineKey {
+            deband_dither: *dither.unwrap_or(&Dither::Disabled),
+            tonemapping: *tonemapping.unwrap_or(&Tonemapping::None),
+        };
+        let pipeline = pipelines.specialize(&pipeline_cache, &upscaling_pipeline, key);
+
+        commands
+            .entity(entity)
+            .insert(ViewTonemappingPipeline(pipeline));
     }
 }
 
```

Comment:
```suggestion
/// Enables a debanding shader that applies dithering to mitigate color banding in the final image.
```

Might be nice to add a docstring, as this is a public component used for toggling the feature.